pulp_ansible/app/migrations/0050_crossrepositorycollectionversionindex.py

```diff
@@ -4,6 +4,13 @@
 import django.db.models.deletion
 
 
+from pulp_ansible.app.tasks.collectionversion_index import rebuild_index
+
+
+def force_build(*args, **kwargs):
+    rebuild_index()
```

Comment:
Don't import functions from the app in the migration. This migration has to be specific to this exact moment in time, and the schema for the index can change over time time (which would mean updates to `rebuild_index()`). You'll have to copy the code for `rebuild_index()` into the migration file.

Reply:
@jctanner not in this case. This is pretty standard for migrations. If you look back through our migrations you'll see this crop up a lot

Reply:
Here's one example: https://github.com/pulp/pulp_ansible/blob/main/pulp_ansible/app/migrations/0035_deprecation_content.py#L68

You'd normally use the `add_and_remove` task for this, but that can't be imported during the migration because it can change over time, so the logic has to be rebuilt